admin/policies/V1.8.0__initial_policy_document.sql

```diff
@@ -50,12 +31,6 @@ ALTER USER thomasyu888 SET TAG ACCOUNT_TYPE = 'service';
 -- SHOW PARAMETERS LIKE 'ENABLE_IDENTIFIER_FIRST_LOGIN' IN ACCOUNT;
 ALTER ACCOUNT SET ENABLE_IDENTIFIER_FIRST_LOGIN = TRUE;
 
--- SHOW PASSWORD POLICIES IN ACCOUNT;
--- SHOW SESSION POLICIES IN ACCOUNT;
--- SHOW AUTHENTICATION POLICIES IN ACCOUNT;
--- SHOW MASKING POLICIES IN ACCOUNT;
--- SHOW NETWORK RULES IN ACCOUNT;
-
 -- Not including CLIENT_TYPES will enable all types for each auth policy
 CREATE AUTHENTICATION POLICY IF NOT EXISTS service_account_authentication_policy
   AUTHENTICATION_METHODS = ('PASSWORD');
```

Comment:
I thought we were doing key-pair authentication for the service accounts - Should this be modified in this PR as well?

Reply:
This is the initial policy document that Tom put together (with some clean up done by myself). I alter this policy in `admin/policies/V1.8.1__alter_authentication_policies.sql`.